examples/filters.go

```diff
@@ -12,7 +12,7 @@ import (
 func main() {
 
 	// login to Array, get groupService instance
-	groupService, err := service.NewNsGroupService("1.1.1.1", "xxx", "xxx", "v1", false)
+	groupService, err := service.NewNimbleGroupService(client.WithHost("1.1.1.1"), client.WithTenantUser("xxx"), client.WithPassword("xxx"), client.WithApiVersion("v1"), client.WithoutWaitForAsyncJobs())
```

Comment:
The WithApiVersion() option should be unnecessary given the proper default. We can remove it from the examples.